src/IO_Interface.hpp

```diff
@@ -0,0 +1,24 @@
+/**
+ * @file
+ * @brief     IOInterface Class
+ * @author    Quinten Perquin
+ * @license   See LICENSE
+ */
+
+#ifndef IO_INTERFACE_HPP
+#define IO_INTERFACE_HPP
+
+#include "Command.hpp"
+
+class IOInterfaceListener;
+
+class IOInterface {
+  public:

```

Comment:
Be consistent in your spacing. *For each indent 4 spaces will be used.*, [wiki](https://github.com/R2D2-2018/R2D2-2018/wiki/Code-standard#indentation)